custom/logistics/templates/logistics/config.html

```diff
@@ -0,0 +1,124 @@
+{% extends 'settings/base_template.html' %}
```

Comment:
Previously we used locations/facility_sync.html template as base template. It's still used in corehq.